tests/yamitranscode.cpp

```diff
@@ -73,24 +73,14 @@ static void print_help(const char* app)
     printf("   --refmode <VP9 Reference frames mode (default 0 last(previous), "
            "gold/alt (previous key frame) | 1 last (previous) gold (one before "
            "last) alt (one before gold)> optional\n");
-}
-
-static VideoRateControl string_to_rc_mode(char *str)
-{
-    VideoRateControl rcMode;
-
-    if (!strcasecmp (str, "CBR"))
-        rcMode = RATE_CONTROL_CBR;
-    else if (!strcasecmp(str, "VBR")) {
-        rcMode = RATE_CONTROL_VBR;
-    }
-    else if (!strcasecmp (str, "CQP"))
-        rcMode = RATE_CONTROL_CQP;
-    else {
-        printf("Unsupport  RC mode\n");
-        rcMode = RATE_CONTROL_NONE;
-    }
-    return rcMode;
+    printf("   --profile optional;\n"
```

Comment:
https://github.com/01org/libyami-utils/pull/86/files/2a161b61b9c1f7c151da7830176fb0e06d3f0874#diff-ca8b349d577083276e5be46dd184960bR72 will show following things are vp9 related. but profile are not vp9 specific.
And also please check other software's setting, HEVCMain sounds little wired.